File: dev-superior-algoritmo-e-logica-de-programacao/c/exercicios/soma_impares.c
#include <stdio.h>

int main() {
    int X, Y, troca, soma;

    printf("Digite dois numeros:\n");
    scanf("%d\n%d", &X, &Y);

    if (X > Y) {
        troca = X;
        X = Y;
        Y = troca;
    }

    soma = 0;
    for (int i = X + 1; i <= Y - 1; i++) {

        if (i % 2 != 0) {
            soma = soma + i;
        }

    }

    printf("SOMA DOS IMPARES = %d\n", soma);

    return 0;
}

File: nelio-alves-algoritmo-e-logica-de-programacao/python/exercicios/soma_impares.py
X: int; Y: int; soma: int; troca: int

print("Digite dois numeros:")
X = int(input())
Y = int(input())

if X > Y:
	troca = X
	X = Y
	Y = troca

soma = 0
for i in range(X + 1, Y):
	if i % 2 != 0:
		soma = soma + i

print(f"SOMA DOS IMPARES = {soma}")